### PR TITLE
adding test to check if rows are skipped when skip_blank_lines is set…

### PR DESCRIPTION
**Pull request by pooja-subramaniam**
… to true in read_csv

xref #22693
test added with xfail decorator
@noatamir @phofl

### CODE REVIEW

**Comment by phofl** [comment]
CI is currently failing, we have to merge main after this is fixed

**Comment by pooja-subramaniam** [comment]
@phofl Could you please merge this if there's no problem in the code?

File: pandas/tests/io/parser/common/test_common_basic.py [hunk 1]
```diff
@@ -862,3 +862,27 @@ def test_read_seek(all_parsers):
...
     tm.assert_frame_equal(actual, expected)
+
+
+@pytest.mark.xfail
+def test_read_csv_skip_blank_rows(all_parsers):
+    # GH22693
+    parser = all_parsers
+    csv_f = StringIO(
+        """A, B, C, D
+                        FOO, 1, 2, 3
```
Comment (phofl) [inline pandas/tests/io/parser/common/test_common_basic.py:873]: Can you move these lines to the beginning of each line?

Comment (phofl) [reply pandas/tests/io/parser/common/test_common_basic.py:873]: otherwise lgtm

File: pandas/tests/io/parser/common/test_common_basic.py [hunk 2]
```diff
@@ -862,3 +862,27 @@ def test_read_seek(all_parsers):
...
+
+@pytest.mark.xfail
+def test_read_csv_skip_blank_rows(all_parsers):
+    # GH22693
+    parser = all_parsers
+    csv_f = StringIO(
+        """A, B, C, D
+    FOO, 1, 2, 3
+    FOO, 4, 5, 6
+    , , ,
```
Comment (phofl) [inline pandas/tests/io/parser/common/test_common_basic.py:875]: Sorry should have been clearer. I meant moving it completely to the left, """ allows this. Please also remove the whitespaces in the blank line completely

**Review by phofl — APPROVED** [review]

**Comment by pooja-subramaniam** [comment]
Thanks for the comments and approval @phofl!

**Comment by phofl** [comment]
Pre-commit is failing, could you check? Otherwise lgtm

**Comment by mroeschke** [comment]
Thanks for the pull request, but it appears to have gone stale. If interested in continuing, please merge in the main branch, address any review comments and/or failing tests, and we can reopen.

**Pull request closed without merging**